FLEDGE.md

```diff
@@ -136,7 +137,10 @@ const myGroup = {
   'maxTrustedBiddingSignalsURLLength' : 10000,
   'userBiddingSignals': {...},
   'ads': [{renderUrl: shoesAd1, sizeGroup: 'group1', ...},
-          {renderUrl: shoesAd2, sizeGroup: 'group2', ...}],
+          {renderUrl: shoesAd2, sizeGroup: 'group2',
+           selectableBuyerAndSellerReportingIds: [ 'deal1', 'deal2', 'deal3' ],
```

Comment:
Nit: Elsewhere (e.g. `trustedBiddingSignalsKeys` above), we don't have spaces immediately following '[' or preceding ']'. Perhaps `selectableBuyerAndSellerReportingIds: ['deal1', 'deal2', 'deal3']`?
```